packages/renderer/src/lib/kube/KubernetesEmptyScreen.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang='ts'>
+import { EmptyScreen } from '@podman-desktop/ui-svelte';
+
+import KubernetesCheckConnection from '/@/lib/ui/KubernetesCheckConnection.svelte';
+
+export let icon: any;
+</script>
+
+<EmptyScreen icon={icon} {...$$props}>
```

Comment:
I am not sure svelte is able to infer properly the props of the `KubernetesEmptyScreen` as an inherit component of EmptyScreen here ? since we just pass down the props, (might be wrong)

In svelte 5 the following is possible

```svelte
<script lang='ts'>
import { EmptyScreen } from '@podman-desktop/ui-svelte';
import KubernetesCheckConnection from '/@/lib/ui/KubernetesCheckConnection.svelte';

let { icon, ...restProps }: ComponentProps<EmptyScreen> = $props();
</script>



<EmptyScreen icon={icon} {...restProps}>...</EmptyScreen>
```

Reply:
The title and message properties are correctly displayed (and the unit tests also assert they are passed to `EmptyScreen`). Or am I misunderstanding your point?

Reply:
`$$props` is a legacy/v4 property

Reply:
explanation: we're running in legacy mode, so it works but if we disable legacy mode, it won't

Reply:
ok, I'll change to Svelte 5, thanks